app/src/App.js

```diff
@@ -1,33 +1,30 @@
-import React from "react";
 import { DrizzleContext } from "@drizzle/react-plugin";
-import { Drizzle, generateStore } from "@drizzle/store";
-
+import { Drizzle } from "@drizzle/store";
+import React from "react";
```

Comment:
Lower-level dependencies should be listed first (i.e. above) higher-level dependencies. So this `import React` line should be at the top of the file.

Reply:
Thanks for this. I didn't realize my eslint was automatically reordering the imports. I had to disable the rule.